bullet_train-api/test/helpers/open_api_helper_test.rb

```diff
@@ -0,0 +1,157 @@
+require "test_helper"
+
+class Api::OpenApiHelperTest < ActiveSupport::TestCase
+  class TestClass
+    include Api::OpenApiHelper
+
+    def initialize
+      @version = "v1"
+    end
+  end
+
```

Comment:
```suggestion

  # TODO: In the future, we can solve this more elegantly than providing
  # a comment here and doing individual asserts. For example, by having a
  # team_attributes.yml fixture and then doing one assert_equal against a
  # string or a `expected_hash.to_yml`.
  #
  # An example of the resulting OpenAPI schema:
  #
```

Added a TODO comment, open for discussion if I'm on the wrong path here.